www-apps/kibana-bin/kibana-bin-5.3.2.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils user
+
+MY_PN="${PN%-bin}"
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Explore and visualize data"
+HOMEPAGE="https://www.elastic.co/products/kibana"
+SRC_URI="amd64? ( https://artifacts.elastic.co/downloads/${MY_PN}/${MY_P}-linux-x86_64.tar.gz )
+	x86? ( https://artifacts.elastic.co/downloads/${MY_PN}/${MY_P}-linux-x86.tar.gz )"
+
+LICENSE="Apache-2.0"
```

Comment:
It seems to bundle NodeJS and a humongous number of modules. You need to include licenses for all of them. However, since it's not a regression, I'm going to just merge it. Nevertheless, consider the issue urgent.